frontend/CLAUDE.md

```
@@ -189,6 +189,281 @@
+          {@html mediaIcons.play}
+        </button>
+      </td>
+    </tr>
+  </tbody>
+</table>
+```
+
+#### Modal Dialogs
+
+**Focus Management**: Modals must trap focus and manage focus properly
+```svelte
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)